Lib/FSW/attitudeControl/SlewAndControlConstrained_init.m

```diff
@@ -6,48 +6,42 @@
 % Author: Cole Morgan
 
 
-controllers = struct;
-controllers.J = simParams.scParams.J;
+% controllers = struct;
+controller1 = struct;
+controller1.J = simParams.scParams.J;
 
 alpha = 45; % angle of the reaction wheels
-A = [cosd(alpha) -cosd(alpha) cosd(alpha) -cosd(alpha);
-    sind(alpha) 0 -sind(alpha) 0;
-    0 -sind(alpha) 0 sind(alpha)];
-controllers.A = A;
+A = [cosd(alpha) 0 -cosd(alpha) 0;
```

Comment:
this guy should be pulled in from the relevant actuator struct -- since I'm including the actuator library I'll make this change when I layer on my code.